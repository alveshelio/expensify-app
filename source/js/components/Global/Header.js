import React from 'react';
import { NavLink } from 'react-router-dom';

const Header = () => (
  <header>
    <h1>Expensify App</h1>
    <nav className='main-navigation'>
      <ul>
        <li><NavLink activeClassName='active' exact to='/'>Dashboard</NavLink></li>
        <li><NavLink activeClassName='active' exact to='/create'>Create Expense</NavLink></li>
        <li><NavLink activeClassName='active' to='/help'>Help</NavLink></li>
      </ul>
    </nav>
  </header>
);

export default Header;

