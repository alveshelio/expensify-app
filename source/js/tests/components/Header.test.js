/* eslint no-undef: "off" */

import React from 'react';
import { shallow } from 'enzyme';

import Header from '../../components/Global/Header';

test('should render Header component correctly', () => {
  const wrapper = shallow(<Header />);

  expect(wrapper).toMatchSnapshot();
});
