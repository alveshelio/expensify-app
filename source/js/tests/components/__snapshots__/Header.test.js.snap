// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header component correctly 1`] = `
<header>
  <h1>
    Expensify App
  </h1>
  <nav
    className="main-navigation"
  >
    <ul>
      <li>
        <NavLink
          activeClassName="active"
          ariaCurrent="true"
          exact={true}
          to="/"
        >
          Dashboard
        </NavLink>
      </li>
      <li>
        <NavLink
          activeClassName="active"
          ariaCurrent="true"
          exact={true}
          to="/create"
        >
          Create Expense
        </NavLink>
      </li>
      <li>
        <NavLink
          activeClassName="active"
          ariaCurrent="true"
          to="/help"
        >
          Help
        </NavLink>
      </li>
    </ul>
  </nav>
</header>
`;
